client/src/styles/Score.css

```diff
@@ -0,0 +1,223 @@
+@keyframes expandHeight {
+  from {
+    height: 1rem;
+  }
+  to {
+    height: var(--final-height);
+  }
+}
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.score-section-wrapper {
+  background-image: linear-gradient(rgba(41, 42, 99, 1), rgba(255, 212, 76, 0)),
+    url("../assets/images/backgroundRegister.png");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-height: 100svh;
```

Comment:
ok: je connaissais pas svh: The svh unit stands for "small viewport height," which is part of the new viewport units introduced to provide better handling of viewport sizes,

Reply:
Thank you